src/document-watcher.js

```diff
@@ -18,29 +18,31 @@ export let flush = function() {};
 
 /**
  * @param {!Element} element
- * @return {!Array<string>}
+ * @return {string}
  */
 function getClasses(element) {
-  let classes = [];
-  if (element.classList) {
-    classes = Array.from(element.classList);
-  } else if (element instanceof window['SVGElement'] && element.hasAttribute('class')) {
-    classes = element.getAttribute('class').split(/\s+/);
+  if (element.classList && element.classList.value !== undefined) {
+    return element.classList.value;
+  } else if (element instanceof window['SVGElement']) {
+    return element.getAttribute('class') || '';
+  } else {
+    return element.className;
```

Comment:
The point of this method is to get non-patched classes that _do_ have the scoping classes in them which are normally hidden?  Can we add a note to that effect?

Also, what is this fallback for (wouldn't it normally be patched)?

Reply:
Per discussion: can be `if (classList.value) { classList.value } else { getAttribute }`